public/components/FeatureAnywhereContextMenu/Container/Container.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import './styles.scss';
+import AssociatedDetectors from '../AssociatedDetectors/containers/AssociatedDetectors';
+
+const Container = ({ startingFlyout, ...props }) => {
+  const { embeddable } = props;
+  const index = [{ label: embeddable?.vis?.data?.indexPattern?.title }];
```

Comment:
we should handle `index` better such that if it doesn't exist, it can be passed down to `Flyout` component with that information. At least specifying the type as `string | undefined` can be helpful here

Reply:
Looking into this, not as easy as `string | undefined` as right now it returns '{ label: any; }[]'

Reply:
Right - or perhaps `{ label: string | undefined }`, or set default as empty string:
```
const index = [{ label: get(embeddable, 'vis.data.indexPattern.title', '') }];
```